src/Compilers/CSharp/Test/Emit/PDB/CSharpDeterministicBuildCompilationTests.cs

```diff
@@ -235,7 +235,7 @@ private static IEnumerable<CSharpCompilationOptions> GetCompilationOptions()
                 delaySign: null,
                 platform: Platform.AnyCpu,
                 generalDiagnosticOption: ReportDiagnostic.Default,
-                warningLevel: 5,
+                warningLevel: 5, // Question: Do you want to use 9999 here instead of 5? or this is explicitly 5 for some reason?
```

Comment:
The comment above says we are trying to provide non-default options on purpose. This warningLevel was originally 4 before this PR. @ryzngard do you care whether this changes as we update the default warning level used by tests?

Reply:
These tests are for correctness in embedded options, so theoretically warning level shouldn't matter for them as long as it doesn't impact the output of the compilation.